From 34e1e3b7615256aef2b42d857b1aadc3acb1183d Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Fri, 14 Feb 2025 13:23:56 +0100
Subject: [PATCH] rust: Enable support for turris1x (powerpc arch)

Enable Enable support for turris1x (powerpc arch) and bump to version 0.2.148,
due to wrong multiple defintions
---
 lang/rust/patches/0003-bump-libc-deps-to-0.2.146.patch | 8 ++++----
 lang/rust/rust-values.mk                               | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/lang/rust/patches/0003-bump-libc-deps-to-0.2.146.patch b/lang/rust/patches/0003-bump-libc-deps-to-0.2.146.patch
index 06ffb70a3..0ca684844 100644
--- a/lang/rust/patches/0003-bump-libc-deps-to-0.2.146.patch
+++ b/lang/rust/patches/0003-bump-libc-deps-to-0.2.146.patch
@@ -7,10 +7,10 @@ This patch bumps all libc dependencies and checksums to 0.2.147, which includes
  [[package]]
  name = "libc"
 -version = "0.2.140"
-+version = "0.2.147"
++version = "0.2.148"
  source = "registry+https://github.com/rust-lang/crates.io-index"
 -checksum = "99227334921fae1a979cf0bfdfcc6b3e5ce376ef57e16fb6fb3ea2ed6095f80c"
-+checksum = "b4668fb0ea861c1df094127ac5f1da3409a82116a4ba74fca2e58ef927159bb3"
++checksum = "9cdc71e17332e86d2e1d38c1f99edcb6288ee11b815fb1a4b049eaa2114d369b"
  
  [[package]]
  name = "lindera"
@@ -21,10 +21,10 @@ This patch bumps all libc dependencies and checksums to 0.2.147, which includes
  [[package]]
  name = "libc"
 -version = "0.2.140"
-+version = "0.2.147"
++version = "0.2.148"
  source = "registry+https://github.com/rust-lang/crates.io-index"
 -checksum = "99227334921fae1a979cf0bfdfcc6b3e5ce376ef57e16fb6fb3ea2ed6095f80c"
-+checksum = "b4668fb0ea861c1df094127ac5f1da3409a82116a4ba74fca2e58ef927159bb3"
++checksum = "9cdc71e17332e86d2e1d38c1f99edcb6288ee11b815fb1a4b049eaa2114d369b"
  
  [[package]]
  name = "libffi"
diff --git a/lang/rust/rust-values.mk b/lang/rust/rust-values.mk
index 035a4e077..a7311d8d7 100644
--- a/lang/rust/rust-values.mk
+++ b/lang/rust/rust-values.mk
@@ -69,7 +69,7 @@ ifeq ($(ARCH),aarch64)
 endif
 
 # Support only a subset for now.
-RUST_ARCH_DEPENDS:=@(aarch64||arm||i386||i686||mips||mipsel||mips64||mips64el||mipsel||powerpc64||riscv64||x86_64)
+RUST_ARCH_DEPENDS:=@(aarch64||arm||i386||i686||mips||mipsel||mips64||mips64el||mipsel||powerpc||powerpc64||riscv64||x86_64)
 
 ifneq ($(CONFIG_RUST_SCCACHE),)
   RUST_SCCACHE_DIR:=$(if $(call qstrip,$(CONFIG_RUST_SCCACHE_DIR)),$(call qstrip,$(CONFIG_RUST_SCCACHE_DIR)),$(TOPDIR)/.sccache)
-- 
2.48.1

